### ISSUE TITLE
Error when saving results as spdx-rdf

### ISSUE THREAD

**Issue by majurg** [nexB/scancode-toolkit#474]
When running the command: `scancode -clip -f spdx-rdf samples/ ~/out.rdf`, I encounter the following error and stack trace:

```
Traceback (most recent call last):
  File "/Users/sesser/Documents/src/scancode-toolkit/bin/scancode", line 11, in <module>
    load_entry_point('scancode-toolkit', 'console_scripts', 'scancode')()
  File "/Users/sesser/Documents/src/scancode-toolkit/lib/python2.7/site-packages/click/core.py", line 664, in __call__
    return self.main(*args, **kwargs)
  File "/Users/sesser/Documents/src/scancode-toolkit/src/scancode/utils.py", line 64, in main
    standalone_mode=standalone_mode, **extra)
  File "/Users/sesser/Documents/src/scancode-toolkit/lib/python2.7/site-packages/click/core.py", line 644, in main
    rv = self.invoke(ctx)
  File "/Users/sesser/Documents/src/scancode-toolkit/lib/python2.7/site-packages/click/core.py", line 837, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/sesser/Documents/src/scancode-toolkit/lib/python2.7/site-packages/click/core.py", line 464, in invoke
    return callback(*args, **kwargs)
  File "/Users/sesser/Documents/src/scancode-toolkit/src/scancode/cli.py", line 289, in scancode
    save_results(files_count, results, format, input, output_file)
  File "/Users/sesser/Documents/src/scancode-toolkit/src/scancode/cli.py", line 735, in save_results
    write_document(doc, str_buffer)
  File "/Users/sesser/Documents/src/scancode-toolkit/lib/python2.7/site-packages/spdx/writers/rdf.py", line 513, in write_document
    writer.write()
  File "/Users/sesser/Documents/src/scancode-toolkit/lib/python2.7/site-packages/spdx/writers/rdf.py", line 491, in write
    files = self.files()
  File "/Users/sesser/Documents/src/scancode-toolkit/lib/python2.7/site-packages/spdx/writers/rdf.py", line 237, in files
    return map(self.create_file_node, self.document.files)
  File "/Users/sesser/Documents/src/scancode-toolkit/lib/python2.7/site-packages/spdx/writers/rdf.py", line 211, in create_file_node
    license_info_nodes = map(self.license_or_special, doc_file.licenses_in_file)
  File "/Users/sesser/Documents/src/scancode-toolkit/lib/python2.7/site-packages/spdx/writers/rdf.py", line 171, in license_or_special
    return self.create_license_node(lic)
  File "/Users/sesser/Documents/src/scancode-toolkit/lib/python2.7/site-packages/spdx/writers/rdf.py", line 159, in create_license_node
    return self.create_license_helper(lic)
  File "/Users/sesser/Documents/src/scancode-toolkit/lib/python2.7/site-packages/spdx/writers/rdf.py", line 121, in create_license_helper
    raise InvalidDocumentError('Missing extracted license: {0}'.format(lic.identifier))
spdx.writers.tagvalue.InvalidDocumentError: Missing extracted license: LicenseRef-jboss-eula
```

**Comment by pombredanne** [comment]
@sschuberth Would you know where is the problem for this?

**Comment by sschuberth** [comment]
Out of my head, no, sorry. So far I've only used *spdx-tv* output. But I'm wondering why the exception is "spdx.writers.*tagvalue*.InvalidDocumentError" when the output is RDF.

**Comment by pombredanne** [comment]
This is fixed in #637
